test/integration/server/helpers/helpers.go

```diff
@@ -94,40 +97,50 @@ func NewGithubClient(ctx context.Context, token string) *ghAPI.Client {
 	return ghAPI.NewClient(tc)
 }
 
-const InAppRoot = ".wego"
-const ExternalConfigRoot = ""
+const InAppRoot = ".weave-gitops"
+const ExternalConfigRoot = ".weave-gitops"
 
 func wegoPath(root, p string) string {
 	return filepath.Join(root, p)
 }
 
-func appPath(root, name string) string {
-	return wegoPath(root, filepath.Join("apps", name, "app.yaml"))
+func appYamlPath(root, name string) string {
+	return appPath(root, name, "app.yaml")
 }
 
-func clusterPath(root, appName, clusterName, filename string) string {
-	return wegoPath(root, filepath.Join("targets", clusterName, appName, filename))
+func automationPath(root, appName string) string {
+	return appPath(root, appName, fmt.Sprintf("%s-gitops-deploy.yaml", appName))
 }
 
-func automationPath(root, appName, clusterName string) string {
-	return clusterPath(root, appName, clusterName, fmt.Sprintf("%s-gitops-deploy.yaml", appName))
+func sourcePath(root, appName string) string {
```

Comment:
It probably makes sense to migrate these to the new ones that I saw in the `/automation` package. Not necessary for this PR, I will create a follow up issue.

Reply:
Yeah, I wasn't sure about that. I've heard people complain in the past that using the same definitions as the ones used to generate can mask a bug. I'm mostly not worried about that but I decided not to make that change yet anyway.

Reply:
Interesting take. I think I like that, since updating this test would indicate a breaking change for the user.